### DISCUSSION TITLE
ScriptReference/RenderTexture

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#234] [Notes]
# ScriptReference/RenderTexture

https://docs.unity3d.com/ScriptReference/RenderTexture.html

**Comment by mvi** [comment]
Be careful to use one of the constructors that takes parameters (i.e. do not use the parameterless constructor with object initializers) because there is a behavioural difference between the parameterless and parametered constructors when it comes to face flipping due to an internal flag that gets set when the parametered constructor is called. If you encounter face flipping issues it may well be because you're calling the wrong constructor.

**Comment by docscomments** [comment]
**Comment by `@Juskelis`:**  

Keep in mind that when you "new" a RenderTexture, on some platforms it does not clear out the memory that the texture uses, which can lead to weird artifacts. In order to clear it out, you need to use GL.Clear.
